purchases/build.gradle

```diff
@@ -31,55 +32,23 @@ dependencies {
     testImplementation 'org.assertj:assertj-core:3.13.2'
 }
 
-apply plugin: 'com.vanniktech.maven.publish'
```

Comment:
this was the original problem: we still had dokka set up for the `purchases` module, but a lot of stuff lives in `public` now